Solutions/Corvus.Monitoring.ApplicationInsights.Specs/AiOperationsInstrumentationSpecs.cs

```diff
@@ -118,20 +118,22 @@ public void WhenOperationWithPostStartPropertiesAndMetricsAddedSeparatelyFinishe
         [Test]
         public void WhenChildOperationFinishesTelemetryIncludesParentId()
         {
-            string testActivityId = Activity.Current.Id;
-            string parentOpActivityId;
-            string childOpActivityId;
+            string? testActivityId = Activity.Current?.Id;
+            string? parentOpActivityId;
+            string? childOpActivityId;
             using (this.Ai.OperationsInstrumentation.StartOperation("ParentOp"))
             {
-                parentOpActivityId = Activity.Current.Id;
+                parentOpActivityId = Activity.Current?.Id;
```

Comment:
Again, if this turns out to be null, it means that the test itself is broken. So consider either a) also declaring these variables as just `string` and using the `?? throw` technique, or alternatively add wording to the `Assert.IsNotNull`s that check this later to include a messages indicating that failure of those particular asserts does not indicate something wrong with the code under test, it indicates something wrong in the test setup.